packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -160,7 +172,29 @@ export class Stepper implements LocalizedComponent, T9nComponent {
   render(): VNode {
     return (
       <Host aria-label={this.messages.label} role="region">
-        <slot onSlotchange={this.handleDefaultSlotChange} />
+        <div
+          class={{ container: true, [CSS.singleView]: this.singleViewMode }}
+          ref={this.setContainerEl}
+        >
+          {this.singleViewMode && (
+            <div class={{ [CSS.stepBarContainer]: true }}>
+              {this.items.map((item, index) => (
+                <StepBar
+                  disabled={item.disabled && index !== this.currentActivePosition}
+                  isActive={index === this.currentActivePosition}
```

Comment:
do you need the `isActive` or could it just be `active`?

Reply:
it can be `active`, renamed `isError` and `isComplete` too.